File: EGIS_Management/Domain/Entities/toims/EstimatedRevenue.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Entities
{
    public class EstimatedRevenue
    {
        [Key]
        public int Id { get; set; }
        public string Code { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public double Amount { get; set; }
        public int SetupYear { get; set; }

    }
}

File: SPA19/EGIS/src/app/toims/models/estimatedrevenue.ts
export interface IEstimatedRevenue {
    id: number;
    code : number;
    description : string;
    amount : number;
    setupyear : number;

 }

 export type createERDto =Omit<IEstimatedRevenue,'code'|'description'|'amount'|'setupyear'>;